browser/gui/app.cpp

```diff
@@ -353,6 +366,15 @@ void App::run_dom_widget() const {
     ImGui::End();
 }
 
+void App::run_stylesheet_widget() const {
+    ImGui::SetNextWindowPos(
+            ImVec2(0, 70 * static_cast<float>(scale_) + window_.getSize().y / 2.f), ImGuiCond_FirstUseEver);
```

Comment:
Suggestions are welcome for how to handle the y-position here.

Reply:
All of the positions and sizes here are very hacked together. :(

Maybe we can do something cleaner using the imgui tab API? Would reduce the number of windows you end up with by default too. If it works well enough, we could even switch to the tab/window docking branch of imgui if it works well enough.

Reply:
Yeah, that sounds good. I'll leave this as it is for now.